client/src/pages/LoginSignup.js

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import { makeStyles } from '@material-ui/core/styles';
+import { Grid, Hidden, Box } from "@material-ui/core";
+import { RedirectButton } from "../LoginSignupComponents";
+import LoginForm from "./LoginForm";
+import SignupForm from "./SignupForm";
+import { themes } from "../themes/LoginSignupThemes";
+
+import { Route, Switch } from "react-router-dom";
+
+import image from "../images/ddb3f7c7b2544f7f1c636f0270f032276c911f02.png"
+
+const useStyles = makeStyles({
+    root: {
+        minHeight: "100vh"
+    },
+    test: {
+    },
+    bg: {
+        // Might need to change url for deployment
+        backgroundImage: `url(${image})`,
+        backgroundRepeat: "no-repeat",
+        backgroundPosition: "center",
+        backgroundSize: "cover 100%"
+    }
+});
```

Comment:
let's do styling like this for every component instead of having a bigger styles sheet called `themes`